apps/freelance/src/app/app.tsx

```diff
@@ -1,25 +1,21 @@
-import styled from 'styled-components';
+import { StyledApp } from './app.styled';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
-import ExamplePage from '../pages/example-page/example-page';
 import store from '../redux/example-store';
```

Comment:
absolute path

Reply:
This was an import removed from the previous commit. But I have refactored ```tsconfig.json``` file and all imports with redux and pages will be absolute path from now on.
I have also refactored ```app.tsx``` and ```main.tsx```. ```<Provider store={store}``` is now inside ```main.tsx```.